test/suite/test_strerror01.py

```diff
@@ -36,6 +36,24 @@
 # test_strerror01.py
 #     Test generation of sub-level error codes when using calling strerror.
 class test_strerror(wttest.WiredTigerTestCase, suite_subprocess):
+    sub_errors = [
+        (-32000, "WT_NONE: No additional context"),
```

Comment:
I would prefer to use the #define error codes themselves. E.g. wiredtiger.WT_NONE

Reply:
Fixed